README.rst

```diff
@@ -156,6 +156,48 @@ To change default order use:
     local t = require('luatest')
     t.configure({shuffle = 'group'})
 
+---------------------------------
+Preloaded hooks
+---------------------------------
+Preloaded hooks extend base hooks. They behave like the pytest fixture with the ``autouse`` parameter.
+
+.. code-block:: lua
+    -- my_helper.lua
+    local hooks = require('luatest.hooks')
+
+    hooks.before_all_preloaded(function() print('start foo') end)
```

Comment:
Shouldn't it be `before_suite_preloaded`?

Reply:
Yep, fixed. `suite -> all -> each` hooks.